### ISSUE TITLE
Proper name validation

### ISSUE THREAD

**Issue by cmhosale** [WGBH/FixIt#3]
Proper names are very hard to validate, and I could see those segments containing them getting caught up in different peoples' interpretations of the correct spelling. If a person or place isn't famous it is hard to know how to spell the name, especially if you move on without searching the internet for it.

I don't know of an inventive way to account for this, I'm just pointing out that it is a complication of the work.

**Comment by pkinchla** [comment]
Agreed. While building the app we frequently noticed several different spellings for _Leonard Lopate_
